pkg/kando/location_delete.go

```diff
@@ -32,43 +29,21 @@ func newLocationDeleteCommand() *cobra.Command {
 		Short: "Delete artifacts from s3-compliant object storage",
 		// TODO: Example invocations
 		RunE: func(c *cobra.Command, args []string) error {
-			return runLocationDelete(c)
+			if err := validateCommandArgs(c); err != nil {
+				return err
+			}
+			dm, err := dataMoverFromCMD(c)
+			if err != nil {
+				return err
+			}
+			dataMover := NewDataMover(dm, "", kopiaSnapshotFlag(c))
```

Comment:
Can we call `NewDataMover()` in `dataMoverFromCMD()` and return dataMover object from `dataMoverFromCMD()`?

Reply:
Same at other occurances

Reply:
Done @PrasadG193  !